components/application-mgt/org.wso2.carbon.identity.application.mgt.ui/src/main/resources/web/application/configure-authentication-flow.jsp

```diff
@@ -858,12 +963,42 @@ var conditionalAuthFunctions = $.parseJSON('<%=availableJsFunctionsJson%>');
 				</div>
 				<div style="clear:both"></div>
                 <div class="buttonRow" style=" margin-top: 10px;">
-                    <input type="button" value="<fmt:message key='button.update.service.provider'/>"
-                           onclick="createAppOnclick();"/>
+                    <input id="createApp" type="button" value="<fmt:message key='button.update.service.provider'/>" />
                     <input type="button" value="<fmt:message key='button.cancel'/>"
                            onclick="javascript:location.href='configure-service-provider.jsp?display=auth_config&spName=<%=Encode.forUriComponent(spName)%>'"/>
                 </div>
             </form>
         </div>
 
+<div style="display: none;">
+	<div id='messagebox-warning' class="messagebox-warning-custom" style="height: auto;">
+		<h3>Prerequisits</h3>
+		<p>Please change the parameters and default steps as required. For more information, refer
+			<a href="https://docs.wso2.com/display/IS560/Conditional+Authentication">Conditional Authentication Docs</a>
```

Comment:
should be 5.7.0 space

Reply:
we change the name to adaptive authentication?

Reply:
@pulasthi7 fixed with d2a0a20
@senthalan This is a template text that we are soon about to populate with data that are from the script template JSON. So lets keep it as it is for the time being.